src/main/java/bookface/model/book/Book.java

```diff
@@ -18,15 +18,33 @@ public class Book {
     private boolean isLoaned = false;
     private Person loanee = null;
 
+    private String loanStatus;
```

Comment:
So to represent this idea of being possibly on loan, we can use an `Optional` for the loanee. So this avoids the possibility of having a NullPointerException and the need for additional `isLoaned` and `loanStatus` fields.

What do you think?